code/datums/traits/negative.dm

```diff
@@ -154,3 +154,12 @@
 	else if(prob(0.5) && dumb_thing)
 		to_chat(H, "<span class='danger'>You think of a dumb thing you said a long time ago and scream internally.</span>")
 		dumb_thing = FALSE //only once per life
+
+/datum/trait/dnr
+	name = "DNR"
```

Comment:
"DNR" is kind of strange here. I'd recommend something like "Weak Genes" or "Unstable Makeup"